atomic_reactor/plugins/pre_download_remote_source.py

```diff
@@ -58,6 +58,22 @@ def get_remote_source_config(self, session, url, insecure=False):
         response.raise_for_status()
         return response.json()
 
+    def generate_cachito_env_file(self):
+        """
+        Generate cachito.env file with exported environment variables received from
+        cachito request.
+        """
+
+        if self.buildargs:
+            self.log.info('Create %s file with environment variables '
+                          'received from cachito request', CACHITO_ENV_FILENAME)
+
+            abs_path = os.path.join(REMOTE_SOURCE_DIR, CACHITO_ENV_FILENAME)
+            with open(abs_path, 'w') as f:
+                f.write('#!/bin/bash\n')
+                for env_var, value in self.buildargs:
```

Comment:
For env vars which are paths, the values here are already absolute paths, right?

Reply:
Yes, since this moment from orchestrator plugin:
* https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/plugins/pre_resolve_remote_source.py#L133